.circleci/config.yml

```diff
@@ -1,17 +1,17 @@
 env: &env
   environment:
     GRUNTWORK_INSTALLER_VERSION: v0.0.36
-    MODULE_CI_VERSION: v0.33.3
+    MODULE_CI_VERSION: v0.41.1
     MODULE_GCP_CI_VERSION: v0.1.1
     TERRAFORM_VERSION: 1.0.3
     PACKER_VERSION: 1.7.4
     TERRAGRUNT_VERSION: v0.32.3
     OPA_VERSION: v0.33.1
     GO_VERSION: 1.16.3
     GO111MODULE: auto
-    K8S_VERSION: v1.15.0  # Same as EKS
-    MINIKUBE_VERSION: v1.9.2
-    HELM_VERSION: v3.1.1
+    K8S_VERSION: v1.18.20  # Same as EKS
+    MINIKUBE_VERSION: v1.22.0
+    HELM_VERSION: v3.7.2
```

Comment:
I was wondering why not update to helm `v3.8.0` (released 3 days ago)

Reply:
Mostly timing of the release. Updated in db6b708f094e1620f1e54e3c38fedf93e09e1675